src/renderer/services/plugin-manager/plugin-sandbox.js

```diff
@@ -52,7 +52,9 @@ export class PluginSandbox {
     }
 
     return {
-      document
+      document,
```

Comment:
Hm, just tested it and we use the `document` object just for compiling templates. Then we can use `getComponentVM()` for the initial file instead of adding the same sandbox for both.

Reply:
🤔 I'm not sure what you mean. If you've already done the changes to test it, do you want to open a PR with the changes and I take a look?